pkg/integration/signature_verifier_interceptor_test.go

```diff
@@ -1,21 +1,38 @@
-package client
+/*
```

Comment:
:+1:

We could think of some gh action to check if such copyright header is present in all go files.